lib/screens/stake_screen/stake_screen.dart

```diff
@@ -0,0 +1,340 @@
+                _showToast = true;
+                Future.delayed(Duration(seconds: 3), () {
+                  setState(() {
+                    _stakeState = StakesStates.stakeApproved;
+                  });
+                });
+              });},
+              label: 'STAKE',
+              selected: true,
+              textStyle: MyStyles.blackMediumTextStyle),
+        ),
+        crossFadeState: _stakeState == StakesStates.canStake || _stakeState == StakesStates.stakePendingApprove || _stakeState == StakesStates.stakeApproved
+            ? CrossFadeState.showSecond
+            : CrossFadeState.showFirst,
+        duration: Duration(milliseconds: 150));
+  }
+
+  Container _buildTextField() {
+    return Container(
+      decoration: BoxDecoration(
+          borderRadius: BorderRadius.circular(MyStyles.cardRadiusSize),
+          color: Color(MyColors.Button_BG_Black),
+          border: Border.all(color: Colors.white.withOpacity(0.1))),
+      padding: EdgeInsets.symmetric(horizontal: 15),
+      child: TextField(
+        cursorColor: Colors.white,
+        style: MyStyles.lightWhiteMediumTextStyle,
+        decoration: InputDecoration(
+            hintText: '0.00',
+            focusedErrorBorder: InputBorder.none,
+            enabledBorder: InputBorder.none,
+            border: InputBorder.none,
+            disabledBorder: InputBorder.none,
+            focusedBorder: InputBorder.none,
+            suffixIcon: Container(
+              margin: EdgeInsets.all(10),
+              padding: EdgeInsets.symmetric(vertical: 5, horizontal: 8),
+              child: Text(
+                'MAX',
+                style: MyStyles.whiteSmallTextStyle,
+              ),
+              decoration: BoxDecoration(
+                  color: Color(0xFF5BCCBD).withOpacity(0.25),
+                  border: Border.all(color: Colors.white, width: 1.5),
+                  borderRadius: BorderRadius.circular(7)),
+            )),
+      ),
+    );
+  }
+
+  GestureDetector _buildBackButton() {
+    return GestureDetector(
+      onTap: () {}, //TODO: nach verknüpfung back logik hinzufügen
+      child: Row(
+        children: [
+          Icon(
+            Icons.arrow_back_ios_rounded,
+          ),
+          SizedBox(
+            width: 8,
+          ),
+          Text(
+            'BACK',
+            style: MyStyles.whiteMediumTextStyle,
+          )
+        ],
+      ),
+    );
+  }
+
+  Row _buildHeaderAddress() {
+    return Row(
+      children: [
+        Container(
+          padding: const EdgeInsets.symmetric(horizontal: 10, vertical: 5),
+          decoration: BoxDecoration(
+              border: Border.all(color: Color(0xFF61C0BF).withOpacity(0.5)),
+              color: Color(0xFF61C0BF).withOpacity(0.25),
+              borderRadius: BorderRadius.all(Radius.circular(6))),
+          child: Center(
+            child: Text(
+              '${address.substring(0, 4)}...${address.substring(address.length - 2)}',
```

Comment:
Hierfür kannst du in lib/ core/util/crypto_util.dart eine Funktion hinzufügen, die die Adresse kürzt.

- [ ] String showShortenedAddress(String address) => address.substring....